src/test/java/org/opentripplanner/middleware/testUtils/OtpTestUtils.java

```diff
@@ -0,0 +1,206 @@
+package org.opentripplanner.middleware.testUtils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.opentripplanner.middleware.otp.OtpDispatcher;
+import org.opentripplanner.middleware.otp.OtpDispatcherResponse;
+import org.opentripplanner.middleware.otp.response.Itinerary;
+import org.opentripplanner.middleware.otp.response.Leg;
+import org.opentripplanner.middleware.otp.response.OtpResponse;
+import org.opentripplanner.middleware.otp.response.Place;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static org.opentripplanner.middleware.otp.OtpDispatcher.OTP_PLAN_ENDPOINT;
+import static org.opentripplanner.middleware.utils.JsonUtils.logMessageAndHalt;
+import static spark.Service.ignite;
+
+public class OtpTestUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(OtpTestUtils.class);
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    /**
+     * Prevents the mock OTP server being initialized more than once
```

Comment:
Typo
```
Prevents the mock OTP server being initialized...
                            ^
                            from
```

Reply:
Fixed in 695f54c4ffb08e23144a496d611b999c93c86764.